MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -127,7 +127,7 @@ public class NavigationDirections: Directions {
         }
     }
     
-    @available(*, deprecated, renamed: "NavigationDirections.configureRouter(tilesURL:completionHandler:)")
+    @available(*, deprecated, message: "`translationsURL` is no-longer a supported argument. Please use NavigationDirections.configureRouter(tilesURL:completionHandler:)")
```

Comment:
We should remove all these deprecated methods anyways, since we’re preparing for v1.0.

Reply:
Agreed. Will do.